### ISSUE TITLE
Datastore browser?

### ISSUE THREAD

**Issue by afader** [allenai/datastore#10]
Let me first say that I love the datastore and it's helping me out a lot!

Now that the formalities are out of the way...

There is starting to be a lot of data in the datastore. It's hard to browse and understand the meaning of the different groups and files. For example, right now I'm trying to find biology-related text. It's hard to tell who owns the different data, what it is, etc. 

I propose that we have a simple datastore browser that has the following features:
* Github-style README.md files for each datastore entry
* Contact info for each datastore entry
* A simple keyword search over the contact information/README.md files.
* Commit messages and timestamps for each version.

**Comment by afader** [comment]
I just realized that what I want is more like "github for data" and less like "maven central for data".

**Comment by dirkgr** [comment]
This is a cool idea, but a little outside the scope for just something I do on the side. When we do planning for Q2, we need to remember to look at the github issues, so that we'll see this, and put it into the plan.

**Comment by dirkgr** [comment]
@codeviking, this is a total distraction, but ... in principle, would it be possible to write a datastore UI purely in JavaScript? Something that lives in an index.html file at the root of the bucket (which is already mapped to http://{public,private}.datastore.allenai.org), and accesses the bucket directly to read and write metadata? As you know, I'm a big fan of not running servers :-)

**Comment by codeviking** [comment]
@dirkgr Looks like Amazon S3 provides a javascript API and after a very quick high-level scan it appears we've got the functionality we'd need to build a simple browser.

So it seems possible.

**Comment by afader** [comment]
@dirkgr Yeah, I think this is pretty low-priority, maybe something for the future...
